salt/metalk8s/orchestrate/worker_plane.sls

```diff
@@ -0,0 +1,93 @@
+{%- set control_plane_ips = salt.saltutil.runner('mine.get', tgt='*', fun='control_plane_ips') %}
+{%- if pillar['bootstrap_id'] in control_plane_ips.keys() and control_plane_ips[pillar['bootstrap_id']] %}
+{%- set control_plane_ip = control_plane_ips[pillar['bootstrap_id']][0] %}
+{%- else %}
+{%- set control_plane_ip = 'localhost' %}
+{%- endif %}
+
+{% set node = pillar.get('node_name', 'node1') %}
```

Comment:
This is not a reasonable default. If the value isn't provided, this should error out.

Reply:
Right, will remove it

Reply:
fixed